StreamChat.xcodeproj/project.pbxproj

```diff
@@ -32,6 +33,10 @@
 		79280F7A248918FD00CDEB89 /* StarscreamWebSocketEngine.swift in Sources */ = {isa = PBXBuildFile; fileRef = 79280F79248918FD00CDEB89 /* StarscreamWebSocketEngine.swift */; };
 		79280F712487CD2B00CDEB89 /* Atomic.swift in Sources */ = {isa = PBXBuildFile; fileRef = 79280F6F2487CD2B00CDEB89 /* Atomic.swift */; };
 		79280F732487CD3100CDEB89 /* Atomic_tests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 79280F702487CD2B00CDEB89 /* Atomic_tests.swift */; };
+		79280F7D24891B1300CDEB89 /* WebSocketEngine_tests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 79280F7B24891B0F00CDEB89 /* WebSocketEngine_tests.swift */; };
```

Comment:
I kinda like the `TestedEntitiy_tests.swift` naming structure. However, IMHO it should be `TestedEntity_Tests.swift`

Reply:
@b-onc so you'd like it to be for example: `VirtualTime_Tests.swift` and then:
```
class VirtualTime_Tests: XCTestCase {
}
```

Reply:
Exactly

Reply:
I agree with @b-onc and yes.

Reply:
OK, sounds good to me 👍